src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck" + " " + machineType + " " + "started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck" + " " + machineType + " " + "stopped working. ");
+
+    }
+
+    @Override
+    public String toString() {
+        return machineType;

```

Comment:
The `toString` method should return a string that represents the object. Returning only `machineType` is not very informative. Consider including the class name and other relevant information.